### PR TITLE
Create Known_SetUp_Issues.adoc

### PR DESCRIPTION
**Pull request by amayagil**
Document to track common issues when configuring AgD on a Mac machine.

##### SUMMARY

Document to track different issues seen when configuring AgD on a Mac box

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by fridim** [comment]
Thanks @amayagil 

Can you rename the file as `.adoc` and not `.doc` so github can display it correctly ?

**Comment by amayagil** [comment]
Sure, I have renamed it to .adoc.

**Comment by fridim** [comment]
i meant the file itself has the .doc extension and it should be .adoc

**Comment by amayagil** [comment]
Renamed file to .adoc

**Comment by amayagil** [comment]
hi anything else you need from me, @fridim ?

**Comment by fridim** [comment]
Since this is running on a mac, i'd hope someone else with experience with mac would review.

File: docs/Known_SetUp_Issues.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+PLAY RECAP *************************************************************************************************************
+localhost                  : ok=16   changed=2    unreachable=0    failed=1
+----
+
+You probably forgot to create or activate your virtual env, to fix this simply run:
++
+[source, shell]
+----
+$ source /Users/[USER]/.pyenv/versions/openstack-ansible-2.9/bin/activate
```
Comment (wkulhanek) [inline docs/Known_SetUp_Issues.adoc:31]: Pyenv? Why?

File: docs/Known_SetUp_Issues.adoc [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+For further information, please refer to link:openstack doc[https://docs.openstack.org/openstacksdk/latest/user/]
+
+3. After installing openstacksdk, I still get same error when running test playbook.
+
+You may be having version issues. Try the following troubleshooting steps:
++
+[source, shell]
+----
+(openstack-ansible-2.9) sassenach:ansible Cibeles$ /Users/Cibeles/.pyenv/versions/openstack-ansible-2.9/bin/pip list |grep openstack
+DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
```
Comment (wkulhanek) [inline docs/Known_SetUp_Issues.adoc:64]: That's a problem. Should really be using Python3 at this point.

Comment (amayagil) [reply docs/Known_SetUp_Issues.adoc:64]: that's is the only way I got it working, some other errors were found when I tried to use python3, but I could not solve them

File: docs/Known_SetUp_Issues.adoc (hunk 1 above)
Comment (amayagil) [reply docs/Known_SetUp_Issues.adoc:31]: it's on the OSP conf steps, without that, I could not get it working

**Comment by NonyNo3lle** [comment]
@amayagil Merging, please update doc making a note that python2 has been EoL since January 1, 2020 and using Python3 will be the recommended path

**Pull request merged**